appConfig/appConfig.snd1.json

```diff
@@ -97,6 +97,13 @@
             "label": "{{serviceName}}",
             "content_type": "{{keyVaultContentType}}",
             "tags": {}
+        },
+        {
+            "key": "Sentinel",
```

Comment:
Is this a shared key?

Reply:
@kbabigumira It is separate for each application differentiated by label